experiments/long_runs/soil.jl

```diff
@@ -207,6 +207,8 @@ function setup_prob(
         start_date;
         output_writer = nc_writer,
         average_period = :monthly,
+        conservation = true,
+        conservation_period = 10 * Day(1),
```

Comment:
I think this should work too
```suggestion
        conservation_period = Day(10),
```